recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/services/language-sync.te

```diff
@@ -40,6 +40,7 @@ dbus_connect_system_bus(language_sync_t)
 dbus_system_bus_client(language_sync_t)
 dbd_dbus_chat(language_sync_t)
 dev_read_sysfs(language_sync_t)
+dev_read_urand(language_sync_t)
```

Comment:
It's weird for language-sync to call `ln` which needs to read urandom.  Looking at language-sync, it calls /usr/share/xenclient/install-bootloader-images, which calls:
```
IMAGES_DIR="/usr/share/xenclient/bootloader/images"
ln -sfn $IMAGES_DIR/$LANGUAGE /config/boot-images
```